packages/ember-htmlbars/lib/node-managers/component-node-manager.js

```diff
@@ -301,13 +310,13 @@ ComponentNodeManager.prototype.destroy = function() {
 export function createComponent(_component, isAngleBracket, _props, renderNode, env, attrs = {}, proto = _component.proto()) {
   let props = assign({}, _props);
 
+  let snapshot = takeSnapshot(attrs);
```

Comment:
ah thanks...

(would be nice if GH should which commit a specific change in a diff came from)